content/applications/finance/fiscal_localizations/philippines.rst

```diff
@@ -146,3 +144,43 @@ Return)* Jan. 2023 version.
 .. important::
    Odoo cannot generate the 2550Q BIR formatted PDF report directly. It should be used as a
    reference when externally filing the form manually or online.
+
+QAP & SAWT Report
+=================
+
+The **QAP**, also known as the *Quarterly Alphalist of Payees* report and the **SAWT**, also known
+as the *Summary Alphalist of Withholding Tax* report can be viewed and exported (in XLSX format).
+The report can be viewed from :menuselection:`Reporting --> Tax Return`. In the Tax Return,
+switch the report to select :menuselection:`Report: --> SAWT & QAP (PH)`.
+
+The report is split into two sections, which can be accessed from their respective buttons at the
+top:
+
+- | :guilabel:`SAWT` for :abbr:`SAWT (Summary Alphalist of Withholding Tax)` report
+  | All customer invoices with the associated sales witholding taxes applied are
+   shown in this report.
+- | :guilabel:`QAP` for :abbr:`QAP (Quarterly Alphalist of Payees)` report
+  | All vendor bills with the associated purchase witholding taxes applied are shown in this report.
+
+.. image:: philippines/philippines-sawt.png
+   :alt: SAWT & QAP Report
+
+.. important::
+   Odoo cannot generate the DAT files directly. The :guilabel:`Export SAWT & QAP` and
+   :guilabel:`XLSX` buttons export an XLSX file, which can be processed using an *external* tool to
+   convert to the DAT format.
+
+Check Printing
+==============
+
+Philippines check print layout is following the latest
+:abbr:`PCHC (Philippine Clearing House Corporation)` standardised printing format.
```

Comment:
```suggestion
:abbr:`PCHC (Philippine Clearing House Corporation)` standardized printing format.
```
We use the American English spelling with "z" instead of "s" :)